src/components/prelevements/point-exploitations.js

```diff
@@ -70,7 +70,18 @@ const PointExploitations = ({pointPrelevement}) => {
         })
 
         const exploitationsWithVolumes = await Promise.all(promises)
-        setExploitationsWithVolumes([...exploitationsWithVolumes])
+        const orderedExploitationsWithVolumes = exploitationsWithVolumes.sort((a, b) => {
+          if (a.statut === 'En activité' && b.statut !== 'En activité') {
+            return -1
+          }
+
+          if (a.statut !== 'En activité' && b.statut === 'En activité') {
+            return 1
+          }
+
+          return 0
+        })
+        setExploitationsWithVolumes([...orderedExploitationsWithVolumes])
```

Comment:
Il n'est pas utile de déstructurer `orderedExploitationsWithVolumes` car `sort()` modifie déjà le tableau d'origin.

```suggestion
        setExploitationsWithVolumes(orderedExploitationsWithVolumes)
```